tmk_core/common/eeconfig.c

```diff
@@ -45,6 +45,9 @@ void eeconfig_init_quantum(void) {
   eeprom_update_byte(EECONFIG_AUDIO,             0xFF); // On by default
   eeprom_update_dword(EECONFIG_RGBLIGHT,      0);
   eeprom_update_byte(EECONFIG_STENOMODE,      0);
+#ifdef VELOCIKEY_ENABLE
```

Comment:
This should not be ifdef'd.  It should ALWAYS be enabled.  

Otherwise, you will may start with random values.